From 360dfe4b6b17378990daef797a7e6a551c799b98 Mon Sep 17 00:00:00 2001
From: Joachim Wiberg <troglobit@gmail.com>
Date: Sun, 2 Feb 2025 06:34:16 +0100
Subject: [PATCH 04/16] Fix #98: fprintlog_first(), drop superfluous call
Organization: Wires

The call to check_timestamp() is not needed (anymore) already called in
logmsg(), which is the only way of entering this function (today).

Signed-off-by: Joachim Wiberg <troglobit@gmail.com>
---
 src/syslogd.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/syslogd.c b/src/syslogd.c
index f812d9e..6748cc6 100644
--- a/src/syslogd.c
+++ b/src/syslogd.c
@@ -2424,9 +2424,6 @@ static void fprintlog_first(struct filed *f, struct buf_msg *buffer)
 
 	logit("Called fprintlog_first(), ");
 
-	/* Messages generated by syslogd itself may not have a timestamp */
-	check_timestamp(buffer);
-
 	if (f->f_type != F_FORW_SUSP && f->f_type != F_FORW_UNKN) {
 		f->f_time = timer_now();
 		f->f_prevcount = 0;
-- 
2.43.0

